dot/parachain/types/types.go

```diff
@@ -640,6 +644,20 @@ type PoV struct {
 	BlockData BlockData `scale:"1"`
 }
 
+// Hash returns the hash of the PoV
+func (p PoV) Hash() (common.Hash, error) {
+	bytes, err := scale.Marshal(p)
+	if err != nil {
+		return common.Hash{}, fmt.Errorf("marshalling PoV: %w", err)
+	}
+
+	return common.Blake2bHash(bytes)
+}
+
+func (p PoV) Encode() ([]byte, error) {
+	return scale.Marshal(p)
+}
```

Comment:
Why do we need the Encode method?
Can't we directly marshal the POV wherever we need it?

Reply:
Yes, I've removed this.